### ISSUE TITLE
[BUG][NOWEB + WEBJS] - Receive Location in dedicated message - message.location (one time + live updates)

### ISSUE THREAD

**Issue by borsatti2** [allburov/whatsapp-http-api#368]
When someone sends location, only a message.any notification will arrive, but without content

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
@borsatti2 hi!
Do you see the data in path `payload._data.message.locationMessage` and fields `degreesLatitude` and `degreesLongitude`?
![image](https://github.com/devlikeapro/waha/assets/155617407/47722fbc-de1c-4c7e-8d32-a40e8cd2626d)

If not - could you send the payload you get in discord :pray: 


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
I agree that we should add `message.location` for both live and one time location tho, we'll do that

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Tested with `2024.6.2` - no issue with location messages

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
My bad, forgot that we should add message.location

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
